<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="enabled">
   <bool>true</bool>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1000</width>
    <height>900</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>1000</width>
    <height>600</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <layout class="QGridLayout" name="gridLayout">
      <item row="2" column="0">
       <layout class="QGridLayout" name="gridLayout_2">
        <property name="topMargin">
         <number>0</number>
        </property>
        <item row="1" column="1">
         <widget class="QLabel" name="label_1">
          <property name="text">
           <string>J1 Shoulder</string>
          </property>
         </widget>
        </item>
        <item row="7" column="2">
         <widget class="QPushButton" name="pushButton">
          <property name="text">
           <string>Start</string>
          </property>
         </widget>
        </item>
        <item row="4" column="0">
         <widget class="QSpinBox" name="spinBox_4"/>
        </item>
        <item row="7" column="0">
         <widget class="QPushButton" name="jontButton">
          <property name="text">
           <string>Set Joint</string>
          </property>
         </widget>
        </item>
        <item row="2" column="1">
         <widget class="QLabel" name="label_2">
          <property name="text">
           <string>J2 Elbow</string>
          </property>
         </widget>
        </item>
        <item row="0" column="2">
         <widget class="QRadioButton" name="haltButton">
          <property name="enabled">
           <bool>true</bool>
          </property>
          <property name="text">
           <string>Halt in safety zone</string>
          </property>
          <property name="checked">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item row="6" column="2">
         <widget class="QPushButton" name="p5Button">
          <property name="enabled">
           <bool>true</bool>
          </property>
          <property name="text">
           <string>Point 5</string>
          </property>
         </widget>
        </item>
        <item row="4" column="3">
         <widget class="QCheckBox" name="checkBox_3">
          <property name="text">
           <string/>
          </property>
          <property name="checked">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item row="1" column="0">
         <widget class="QSpinBox" name="spinBox_1"/>
        </item>
        <item row="3" column="1">
         <widget class="QLabel" name="label_3">
          <property name="text">
           <string>J3 Wrist 1</string>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QLabel" name="label_0">
          <property name="text">
           <string>J0 Base</string>
          </property>
         </widget>
        </item>
        <item row="3" column="0">
         <widget class="QSpinBox" name="spinBox_3"/>
        </item>
        <item row="2" column="2">
         <widget class="QPushButton" name="p1Button">
          <property name="text">
           <string>Point 1</string>
          </property>
         </widget>
        </item>
        <item row="6" column="3">
         <widget class="QCheckBox" name="checkBox_5">
          <property name="text">
           <string/>
          </property>
          <property name="checked">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item row="1" column="2">
         <widget class="QPushButton" name="zeroButton">
          <property name="text">
           <string>Zero position</string>
          </property>
         </widget>
        </item>
        <item row="3" column="2">
         <widget class="QPushButton" name="p2Button">
          <property name="text">
           <string>Point 2</string>
          </property>
         </widget>
        </item>
        <item row="2" column="3">
         <widget class="QCheckBox" name="checkBox">
          <property name="text">
           <string/>
          </property>
          <property name="checked">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item row="3" column="3">
         <widget class="QCheckBox" name="checkBox_2">
          <property name="text">
           <string/>
          </property>
          <property name="checkable">
           <bool>true</bool>
          </property>
          <property name="checked">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item row="4" column="2">
         <widget class="QPushButton" name="p3Button">
          <property name="text">
           <string>Point 3</string>
          </property>
         </widget>
        </item>
        <item row="5" column="1">
         <widget class="QLabel" name="label_5">
          <property name="text">
           <string>J5 Wrist 3</string>
          </property>
         </widget>
        </item>
        <item row="2" column="0">
         <widget class="QSpinBox" name="spinBox_2"/>
        </item>
        <item row="5" column="0">
         <widget class="QSpinBox" name="spinBox_5"/>
        </item>
        <item row="0" column="0">
         <widget class="QSpinBox" name="spinBox_0"/>
        </item>
        <item row="4" column="1">
         <widget class="QLabel" name="label_4">
          <property name="text">
           <string>J4 Wrist 2</string>
          </property>
         </widget>
        </item>
        <item row="5" column="2">
         <widget class="QPushButton" name="p4Button">
          <property name="text">
           <string>Point 4</string>
          </property>
         </widget>
        </item>
        <item row="5" column="3">
         <widget class="QCheckBox" name="checkBox_4">
          <property name="text">
           <string/>
          </property>
          <property name="checked">
           <bool>true</bool>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item row="1" column="0">
       <spacer name="verticalSpacer">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>40</height>
         </size>
        </property>
       </spacer>
      </item>
      <item row="0" column="0">
       <layout class="QVBoxLayout" name="verticalLayout"/>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QGridLayout" name="gridLayout_3">
      <item row="3" column="2">
       <widget class="QPushButton" name="savePosButton">
        <property name="text">
         <string>Save Position</string>
        </property>
       </widget>
      </item>
      <item row="6" column="2">
       <widget class="QPushButton" name="startButton">
        <property name="text">
         <string>Start</string>
        </property>
       </widget>
      </item>
      <item row="5" column="2">
       <widget class="QTableWidget" name="tableWidget">
        <property name="rowCount">
         <number>10</number>
        </property>
        <property name="columnCount">
         <number>3</number>
        </property>
        <attribute name="horizontalHeaderCascadingSectionResizes">
         <bool>false</bool>
        </attribute>
        <row/>
        <row/>
        <row/>
        <row/>
        <row/>
        <row/>
        <row/>
        <row/>
        <row/>
        <row/>
        <column/>
        <column/>
        <column/>
       </widget>
      </item>
      <item row="2" column="2">
       <widget class="QPushButton" name="loadPosButton">
        <property name="text">
         <string>Load Position</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1000</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
